clients/src/main/java/org/apache/kafka/clients/consumer/internals/ConsumerNetworkThread.java

```diff
@@ -289,10 +289,11 @@ private void closeInternal(final Duration timeout) {
         }
     }
 
+    // Add test to see if poll() is run once with timer of 0
```

Comment:
is this something you intend to do on this PR or separately? Either way, I would suggest we remove the comment and address it in this PR or have a jira to follow-up separately if needed please.

Reply:
I forgot to remove that, removed now